packages/benchmarks/lit-element/stub1/tachometer.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Polymer/tachometer/master/config.schema.json",
+  "sampleSize": 30,
+  "timeout": 0,
+  "benchmarks": [
+    {
+      "url": "lit-element/stub1/stub1.html",
+      "browser": {
+        "name": "chrome",
+        "headless": true
+      },
+      "measurement": {
+        "mode": "performance",
+        "entryName": "render"
+      },
+      "expand": [
+        {
+          "name": "this-change"
+        },
+        {
+          "name": "tip-of-tree",
+          "packageVersions": {
+            "label": "tip-of-tree",
+            "dependencies": {
+              "lit-html": {
+                "kind": "git",
+                "repo": "https://github.com/Polymer/lit-html.git",
+                "ref": "lit-next",
+                "subdir": "packages/lit-html",
+                "setupCommands": [
+                  "npm ci",
+                  "npm run bootstrap",
+                  "npm run build"
+                ]
+              },
+              "lit-element": {
+                "kind": "git",
+                "repo": "https://github.com/Polymer/lit-html.git",
+                "ref": "lit-next",
+                "subdir": "packages/lit-element",
+                "setupCommands": [
+                  "npm ci",
+                  "npm run bootstrap",
+                  "npm run build"
+                ]
+              }
+            }
+          }
+        },
+        {
+          "name": "previous-release",
+          "packageVersions": {
+            "label": "previous-release",
+            "dependencies": {
+              "lit-html": "^1.0.0",
```

Comment:
Do you need this, or will it come along with lit-element?

Reply:
Since `lit-html` is declared in the `package.json` at `^2.0.0-pre.0`, we'd get an error since that's not published, plus it would install two versions even if it was published.